setup.cfg

```diff
@@ -24,3 +24,6 @@ no_implicit_reexport = True
 
 ignore_missing_imports = True
 exclude = .venv|venv|build|docs|tutorial|optuna/storages/_rdb/alembic
+
+[mypy-torch.*]
+follow_imports = skip
```

Comment:
According to [the MyPy official documentation](https://mypy.readthedocs.io/en/stable/running_mypy.html#following-imports), this line has an effect to simply replace any `torch`-related modules with `Any` in type checking.
Probably, it should be mentioned.

```suggestion
# Any `torch`-related modules will be replaced with `typing.Any` at type checking.
follow_imports = skip
```

Reply:
Agreed